helix-core/src/main/java/org/apache/helix/monitoring/mbeans/ParticipantMessageMonitor.java

```diff
@@ -19,10 +19,27 @@
  * under the License.
  */
 
-public class ParticipantMessageMonitor implements ParticipantMessageMonitorMBean {
-  public static final String PARTICIPANT_KEY = "ParticipantName";
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
Question: was this formatted? If yes, then our formatter has problem.

Reply:
Yes, it's formatted by `helix-style-intellij.xml`. What problem do you think we have here?

Reply:
It seems good to me. @dasahcc, it is javax below, not java. If this is what concerns you : )